### ISSUE TITLE
No Clocks Data Available; Radeon RX 7900 GRE [XFX]

### ISSUE THREAD

**Issue by czar** [ilya-zlobintsev/LACT#373]
### Checklist

- [X] The problem is not listed in the [hardware support matrix](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#hardware-support) as a known limitation
- [X] I've included a [debug snapshot](https://github.com/ilya-zlobintsev/LACT?tab=readme-ov-file#reporting-issues) if the issue is related to hardware configuration

### Bug description

I’m interested in trying out this software, but I’m encountering a “No Clocks Data Available” issue. I’m hoping to lower the clock speed, as some users have suggested, to address the crashes I’ve been experiencing while playing Counter-Strike 2.

I’ve attached a debug snapshot for your reference. Thank you for your assistance!

[LACT-sysfs-snapshot-20240911-174041.tar.gz](https://github.com/user-attachments/files/16970699/LACT-sysfs-snapshot-20240911-174041.tar.gz)

### System info

```markdown
- LACT version: 0.5.6-0
- GPU model:  AMD Radeon rx 7900 gre  (XFX brand)
- Kernel version:  Linux version 6.5.0-45-generic (buildd@lcy02-amd64-109) (x86_64-linux-gnu-gcc-12 (Ubuntu 12.3.0-1ubuntu1~22.04) 12.3.0, GNU ld (GNU Binutils for Ubuntu) 2.38) #45~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Jul 15 16:40:02 UTC 2
- Distribution:  Ubuntu 22.04.4 LTS
```

**Comment by ilya-zlobintsev** [comment]
Did you enable OC support? It is required to edit clocks settings.

**Comment by czar** [comment]
> Did you enable OC support? It is required to edit clocks settings.

I tried toggling OC support on and off and rebooting, but I didn't see any difference in the clock data becoming available.

The issue was resolved by first updating the amdgpu firmware located in /lib/firmware/amdgpu/ from [this repository](https://git.kernel.org/pub/scm/linux/kernel/git/firmware/), and then by upgrading to kernel 6.8 from 6.5. The kernel upgrade alone didn’t work until the new firmware was applied.

It looks like the default maximum clock is set to 2415 MHz, whereas my card's specifications state the max is 2395 MHz. Hopefully, this resolves the CS2 issues. Thanks a lot! :)
